Assets/Scripts/Towers/TurretInterface.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface TurretInterface
+{
+    //Function to be called in animation controller to make the turret fire.
```

Comment:
Having pretty much no experience with animation (`.anim`) files, when I read the comment in [`LightningShootable.cs`](https://github.com/SondreHus/HackerSpace_NTNU_SpillProsjekt/blob/d3e0ce38158c84526e8bafaaa798ca4ca0fc9e0c/Assets/Scripts/Towers/LightningShootable.cs#L9), I started looking through the animation controller for the lightning tower.. After way too long, I finally found out that the shooting function was actually called from the animation file - not the animation controller - and so I think it would decrease potential confusion if this comment were changed to something like this: 😊 
```suggestion
    // Called through animation events on `.anim` files to make the turret fire.
```